styleguide.config.js

```diff
@@ -81,6 +81,10 @@ module.exports = {
         {
           name: 'Forms',
           components: 'src/organisms/forms/**/*.js'
+        },
+        {
+          name: 'Tables',
+          components: () => [ 'src/organisms/tables/ComparisonTable/index.js' ]
```

Comment:
@cisacke If this is going to be a Table category, this should probably be a blob with a wildcard so it will pull all the js from the tables directory. Why are you using a function here?